onadata/apps/api/tools.py

```diff
@@ -142,9 +143,12 @@ def create_organization(name, creator):
 
     """
     organization, _created = User.objects.get_or_create(username__iexact=name)
-    organization_profile = OrganizationProfile.objects.create(
-        user=organization, creator=creator
-    )
+    try:
```

Comment:
Why not just `OrganizationProfile.objects.get_or_create()` ? Also, what scenario has to occur for a user created in step 145 to have the profile created before the next line ?